pyproject.toml

```diff
@@ -2,3 +2,88 @@
 # Minimum requirements for the build system to execute.
 requires = ["setuptools>=36.6", "wheel"]
```

Comment:
I think you now need to bump minimum setuptools version to 61 (or even 61.2, as some recommend).

Reply:
Good call. Thanks.